test/server/index.js

```diff
@@ -436,6 +573,35 @@ function initST({ passwordlessConfig } = {}) {
                     }),
                     customAuth0Provider(),
                 ],
+                override: {
+                    apis: (originalImplementation) => {
+                        return {
+                            ...originalImplementation,
```

Comment:
need to change this as well - make sure to not use request interception anywhere

Reply:
Also add passwordless